.travis.yml

```diff
@@ -1,6 +1,4 @@
 language: node_js
-node_js:
-  - 12.16
```

Comment:
Why did we remove this?

Reply:
TravisCI automatically picks up `.nvmrc` now, so we don't need to specify it here.